doc/whats_new.rst

```diff
@@ -21,6 +21,13 @@ Changes
       :class:`nilearn.decomposition.DictLearning` is deprecated and will
       be removed in next two releases. Use `components_img_` instead.
 
+Bug fixes
+---------
+
+    - Fix issues using :func:`nilearn.plotting.plot_connectome` when string is
+      passed in `node_color` display modes left and right hemispheric cuts
+      in the glass brain.
```

Comment:
It seems to me that this list is incomplete. We should add the other bug fixes before the release.